test/utils/test-entry-identity.js

```diff
@@ -0,0 +1,51 @@
+const Log = require('../../src/log')
```

Comment:
Do we still need `test-entry-identity.js` or is it obsolete? I believe we're not using it anywhere anymore.